apps/react-18-tests-v9/cypress-react-18.config.ts

```diff
@@ -8,6 +8,7 @@ const excludedSpecs = [
 
 // Include all tests from this app and the components package
 const specs = [
+  path.resolve('./src/**/*.cy.{tsx,ts}'),
```

Comment:
This enables some tests that were not included. These are the tests that currently live inside the react-18-tests-v9 app.